src/components/wallet/Send.tsx

```diff
@@ -1,38 +1,57 @@
 import React from 'react'
 
-import { Row, Col, Form, Input, Button, Typography } from 'antd'
+import { Row, Form } from 'antd'
 import { Store } from 'antd/lib/form/interface'
+import { useIntl } from 'react-intl'
 
+import Button from '../../components/uielements/button'
+import Input from '../../components/uielements/input'
+import Label from '../../components/uielements/label'
 import AccountSelector from './AccountSelector'
-
-const { Title } = Typography
+import { StyledCol, StyledForm, StyledSubForm, StyledFormItem, StyledSubmitItem, StyledLabel } from './Send.style'
 
 const Send: React.FC = (): JSX.Element => {
+  const intl = useIntl()
+
   const onSubmit = (data: Store) => {
     console.log(data)
   }
+
   return (
     <Row>
-      <Col sm={{ span: 24 }} md={{ span: 16, offset: 4 }} lg={{ span: 12, offset: 6 }}>
-        <Title level={4}>Send Funds</Title>
+      <StyledCol span={24}>
         <AccountSelector />
```

Comment:
@FrozenIce0617 Could you check `AccountSelector`, pls? It seems it's not possible to select another value of it.